.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💬 Discord
+    url: https://discord.com/invite/8KtRkjq2Q4
```

Comment:
Is this link expire-free?

Reply:
Yes. That's the link you created.

Reply:
![image](https://github.com/user-attachments/assets/438fd946-be1f-4df1-ae8c-172a8275d794)